src/main/proto/th2_grpc_rpt_data_provider/rpt_data_provider_template.proto

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+import "th2_grpc_common/common.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/any.proto";
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/empty.proto";
+
+option java_multiple_files = true;
+option java_package = "com.exactpro.th2.rptdataprovider.grpc";
+
+service RptDataProvider {
+
+  rpc getEvent (EventID) returns (RptEvent) {
+  }
+
+  rpc getMessage (MessageID) returns (RptMessage) {
+  }
+
+  rpc getMessageStreams (google.protobuf.Empty) returns (StringList) {
+  }
+
+  rpc searchSseMessages (GrpcMessageSearchRequest) returns (stream StreamResponse) {
+  }
+
+  rpc searchSseEvents (GrpcEventSearchRequest) returns (stream StreamResponse) {
+  }
+
+  rpc getMessagesFilters (google.protobuf.Empty) returns (ListFilterName) {
+  }
+
+  rpc getEventsFilters (google.protobuf.Empty) returns (ListFilterName) {
+  }
+
+  rpc getEventFilterInfo (FilterName) returns (FilterInfo) {
+  }
+
+  rpc getMessageFilterInfo (FilterName) returns (FilterInfo) {
+  }
+
+  rpc  matchEvent (MatchRequest) returns (IsMatched) {
+  }
+
+  rpc  matchMessage (MatchRequest) returns (IsMatched) {
+  }
+}
+
+message RptEvent {

```

Comment:
Please choose a name without `Rpt`, because we want to rename rpt-data-provider to data-provider